### DISCUSSION TITLE
docker-compose类型的靶场环境如何使用？

### DISCUSSION THREAD

**Discussion by nsvicp** [fofapro/vulfocus#318] [Q&A] [answered]
![4557f04ba07b5d77e5ef6626e8ff811](https://user-images.githubusercontent.com/45115846/199461494-786a3080-eccd-4a4e-be25-ee70736bb852.jpg)

请问docker-compose类型的靶场环境如何使用？

以及如果我需要导入vulhub的docker-compose.yml文件失败怎么处理。
![image](https://user-images.githubusercontent.com/45115846/199462012-9ea1efbe-46f4-4e60-94d6-a2a473ce6bec.png)
![image](https://user-images.githubusercontent.com/45115846/199462088-1bbc816b-c49f-44bb-8513-835e37dd2588.png)

**Comment by alilovetaozi** [comment, marked as answer]
https://github.com/fofapro/vulfocus/issues/317
